Makefile

```diff
@@ -329,7 +329,7 @@ endif
 
 $(BUILD_DIR)/windows-amd64/crc-windows-amd64.msi: msidir
 	candle.exe -arch x64 -ext WixUtilExtension -o $(PACKAGE_DIR)/msi/ $(PACKAGE_DIR)/msi/*.wxs
-	cd $(PACKAGE_DIR)/msi && light.exe -ext WixUIExtension -ext WixUtilExtension -sacl -spdb -sice:ICE61 -sice:ICE69 -out ../../../$@ *.wixobj
+	cd $(PACKAGE_DIR)/msi && light.exe -ext WixUIExtension -ext WixUtilExtension -sacl -spdb -sice:ICE61 -sice:ICE69 -out $(BUILD_DIR)/windows-amd64/$@ *.wixobj
```

Comment:
I'm a bit confused by this change, I'd expect `$@` to expand to the target name, which is `$(BUILD_DIR)/windows-amd64/crc-windows-amd64.msi` here

```
$ cat Makefile 
all: /tmp/makefile-test

/tmp/makefile-test:
	echo "$@"
	touch "$@"

$ make
echo "/tmp/makefile-test"
/tmp/makefile-test
touch "/tmp/makefile-test"

Reply:
@cfergeau this is something I asked to @anjannath, will look more closely

Reply:
Ahh, my bad, i thought its only the last part of the target and not the whole target, and since $(BUILD_DIR) is `out/`, to have the full path we needed the `../../`